code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -219,56 +219,31 @@
 	if(method == REAGENT_TOUCH)
 		if(ishuman(M))
 			var/mob/living/carbon/human/victim = M
-			var/mouth_covered = 0
-			var/eyes_covered = 0
+			var/eyes_covered = FALSE
 			var/obj/item/safe_thing = null
-			if(victim.wear_mask)
-				if(victim.wear_mask.flags_cover & MASKCOVERSEYES)
-					eyes_covered = 1
-					safe_thing = victim.wear_mask
-				if(victim.wear_mask.flags_cover & MASKCOVERSMOUTH)
-					mouth_covered = 1
-					safe_thing = victim.wear_mask
-			if(victim.head)
-				if(victim.head.flags_cover & MASKCOVERSEYES)
-					eyes_covered = 1
-					safe_thing = victim.head
-				if(victim.head.flags_cover & MASKCOVERSMOUTH)
-					mouth_covered = 1
-					safe_thing = victim.head
+			if(victim.wear_mask && victim.wear_mask.flags_cover & MASKCOVERSEYES)
+				eyes_covered = TRUE
+				safe_thing = victim.wear_mask
+			if(victim.head && victim.head.flags_cover & MASKCOVERSEYES)
+				eyes_covered = TRUE
+				safe_thing = victim.head
 			if(victim.glasses)
-				eyes_covered = 1
+				eyes_covered = TRUE
 				if(!safe_thing)
 					safe_thing = victim.glasses
-			if(eyes_covered && mouth_covered)
+
+			if(eyes_covered && victim.internal)
```

Comment:
Please mention in the changelog too, that having a mask no longer counteracts pepperspray